src/php/Munge.php

```diff
@@ -58,6 +58,7 @@ public static function decodeNs(string $s): string
         return self::encodeWithMap($s, array_flip(self::$mungeNsMapping));
     }
 
+    /** @param (array-key|mixed)[] $mapping */
     private static function encodeWithMap(string $s, $mapping): string
```

Comment:
Why not add `array $mapping`?

Reply:
This syntax is autogenerated by psalm when you run the auto fixer (from psalm suggestions). Anyway, I would also prefer to use `array` and/or even better the array types.

Reply:
I agree, I thought about using something like: `array<string, string>` but I didn't know what key and value types were, so I prefer to use the psalm standard.
I will investigate later and replace by the good ones.
BTW, @jenshaase do you know what is the shape of the array?